tests/ha/check_after_reboot.pm

```diff
@@ -68,8 +68,17 @@ sub run {
         assert_screen 'iscsi-client-overview-service-tab', $default_timeout;
         send_key 'alt-v';
         wait_still_screen 3;
-        assert_screen 'iscsi-client-target-connected', $default_timeout;
-        send_key 'alt-c';
+        assert_screen 'iscsi-client-target-list', $default_timeout;
+
+        if (!check_screen('iscsi-client-target-connected', $default_timeout)) {
```

Comment:
`default_timeout` is too big for `check_screen`, it will wait whole timeout. Smaller timeout or use `match_has_tag`
```
[2022-05-18T13:54:00.002615+02:00] [debug] >>> testapi::_check_backend_response: match=iscsi-client-target-connected timed out after 120 (check_screen)
[2022-05-18T13:54:00.019287+02:00] [debug] tests/ha/check_after_reboot.pm:75 called testapi::send_key
```

Reply:
Yep, makes sense, thanks. I will try 'match_has_tag' and do another VR.
Just a question : If I used 'check_screen' with smaller timeout. What would be a good value to use? I mean beside picking a static number of seconds?

Reply:
I don't know exactly, I would use probably 5 sec.